### PR TITLE
FIX: Respect visibility in the LegendHighlighter

### PR DESCRIPTION
**Pull request by jwiggins**
Before, the `LegendHighlighter` tool would respond to mouse clicks even if the legend was not visible.

### CODE REVIEW

File: chaco/tools/legend_highlighter.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import operator
 
-import six
```
Comment (jwiggins) [inline chaco/tools/legend_highlighter.py]: Unrelated cleanup. This import was not used at all in the file

**Comment by corranwebster** [comment]
LGTM, although is this the only place where there is this sort of problem?  I'm not sure we should do something as drastic as not dispatching events to non-visible components in Enable, but it might be worth a quick audit of other tools.

**Comment by jwiggins** [comment]
Thanks for the quick review. Shall I create an issue for the audit of the other tools?

**Comment by corranwebster** [comment]
Yes, that would be a good idea.
